packages/rich-text/src/plugins/Text/__tests__/createTextPlugin.test.tsx

```diff
@@ -28,7 +28,7 @@ describe('delete backward', () => {
     },
     {
       title:
-        'deletes the empty paragraph at the beginning of the RTE followed by another paragraph',
+        'does not delete the empty paragraph at the beginning of the RTE followed by another paragraph',
```

Comment:
No other Rich Text I tested against has this behaviour, I argue that we should do the opposite and not remove the empty paragraph at the beginning